content/webapp/pages/landing-page.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+import type { Context } from 'next';
+import { Component } from 'react';
+import PageLayout from '@weco/common/views/components/PageLayout/PageLayout';
+import ContentPage from '@weco/common/views/components/ContentPage/ContentPage';
+import LandingBody from '@weco/common/views/components/LandingBody/LandingBody';
+import HTMLDate from '@weco/common/views/components/HTMLDate/HTMLDate';
+import HeaderBackground from '@weco/common/views/components/HeaderBackground/HeaderBackground';
+import PageHeader from '@weco/common/views/components/PageHeader/PageHeader';
+import VideoEmbed from '@weco/common/views/components/VideoEmbed/VideoEmbed';
+import { UiImage } from '@weco/common/views/components/Images/Images';
+import { convertImageUri } from '@weco/common/utils/convert-image-uri';
+import { getPage } from '@weco/common/services/prismic/pages';
+import { contentLd } from '@weco/common/utils/json-ld';
+import type { Page as PageType } from '@weco/common/model/pages';
+
+type Props = {|
+  page: PageType,
+|};
+
+const backgroundTexture =
+  'https://wellcomecollection.cdn.prismic.io/wellcomecollection%2F9154df28-e179-47c0-8d41-db0b74969153_wc+brand+backgrounds+2_pattern+2+colour+1.svg';
+export class Page extends Component<Props> {
+  static getInitialProps = async (ctx: Context) => {
+    const { id, memoizedPrismic } = ctx.query;
+    ctx.query.memoizedPrismic = undefined; // Once we've got memoizedPrismic, we need to remove it before making requests - otherwise we hit circular object issues with JSON.stringify
```

Comment:
Is this still required? I thought you had a PR that deleted it higher up the chain?

Reply:
Good point, you're right